migrations/2018-06-22-01-member-invite-confirmed.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Set the invite_confirmed field on all members.
+ */
+
+import * as path from "path";
+
+import * as admin from "firebase-admin";
+import {
+  Firestore,
+  QueryDocumentSnapshot,
+  DocumentSnapshot
+} from "@google-cloud/firestore";
+
+import { getDb } from "./helpers";
+import { request } from "http";
+
+async function setIsInviteConfirmed(db: Firestore) {
+  const members = (await db.collection("members").get()).docs;
+  const inviteOperations = (await db
+    .collection("operations")
+    .where("op_code", "==", "REQUEST_INVITE")
+    .get()).docs;
+  const trustOperations = (await db
+    .collection("operations")
+    .where("op_code", "==", "TRUST")
+    .get()).docs;
+
+  for (const i in members) {
+    const member = members[i];
+
+    const inviteOps = inviteOperations.filter(
+      op => op.get("creator_uid") === member.id
+    );
+    if (inviteOps.length < 1) {
+      console.log(
+        "member has zero request invite ops",
+        member.id,
+        inviteOps.length
+      );
+      continue;
+    } else if (inviteOps.length > 1) {
```

Comment:
Are we just skipping these? why?

Reply:
Actually, jk, I can't remove it since the "RAHA" user has never requested an invite and should be skipped for now. No one's requested multiple invites though.